python/tests/test_highlevel.py

```diff
@@ -1441,6 +1441,33 @@ def verify_pairwise_diversity(self, ts):
     def test_pairwise_diversity(self, ts):
         self.verify_pairwise_diversity(ts)
 
+    def test_bad_node_order(self):
+        ts = tskit.TableCollection(1).tree_sequence()
+        for order in ["abc", 0, 1, False]:
```

Comment:
This is a pytest.parametrize pattern - generally if you're using a loop in a test like this, change it to a parametrize.

Reply:
Oh yes, of course.